modules/css/index.scss

```diff
@@ -11,7 +11,7 @@ $wdc-system-icons-path: '../../node_modules/@workday/canvas-system-icons-web/dis
 @import '~@workday/canvas-kit-css-form-field/index.scss';
 @import '~@workday/canvas-kit-css-icon/index.scss';
 @import '~@workday/canvas-kit-css-layout/index.scss';
-@import '~@workday/canvas-kit-css-loading-animation/index.scss';
+@import '~@workday/canvas-kit-css-loading-dots/index.scss';
```

Comment:
do we care about upgrade this? Would this cause a package bump?

Reply:
This could add to the confusion we're trying to remove, if we don't change it. The package already bumps, so nothing changes there. It started with fixing the Storybook paths (Loading Animation -> CSS) and I realized this would be a loose end if we don't update the package name.

Reply:
ok cool thanks!